plugins/main/bridge/bridge.go

```diff
@@ -32,14 +32,21 @@ import (
 	"github.com/vishvananda/netlink"
 )
 
-const defaultBrName = "cni0"
+const (
+	defaultBrName           = "cni0"
```

Comment:
AFAICT the constants below are not used anymore. Please revert changes to this section to the previous state.